### DISCUSSION TITLE
PUSH problem from my windows 10

### DISCUSSION THREAD

**Discussion by ealalin** [community/community#78311] [New to GitHub]
From my windows 10 pc .
I want to put my python program on gituh . this first trial is published from my directory
C:\>cd c:/learning_log
here are the following command done 
c:\learning_log>git init
Reinitialized existing Git repository in C:/learning_log/.git/

c:\learning_log>git add .

c:\learning_log>git commit -m "initial commit"
On branch main
nothing to commit, working tree clean

c:\learning_log>git remote add origin https://github.com/ealalin/learning_log.git
error: remote origin already exists.

c:\learning_log>git remote -v
origin  https://github.com/ealalin/learning_log.git (fetch)
origin  https://github.com/ealalin/learning_log.git (push)

c:\learning_log>git push -u origin main
remote: Permission to ealalin/learning_log.git denied to Ealalin02.
fatal: unable to access 'https://github.com/ealalin/learning_log.git/': The requested URL returned error: 403
Why  it's denied to Ealalin02 ?  I 'm logged in as ealalin ?
I can't remove ealalin02 because is use some where else.

**Comment by jge162** [comment]
Run `git config --list` to view the Git configuration.

- Look for entries like user.name and user.email and ensure they match the credentials of your ealalin GitHub account.
- if not they fix them.
